### PR TITLE
Set config variables in home.html and read them for production deploys

### PR DESCRIPTION
**Pull request by betamax**

### CODE REVIEW

File: backend/config/settings/production.py [hunk 1]
```diff
@@ -222,3 +222,8 @@
 )
 
 DJANGO_REST_MULTITOKENAUTH_RESET_TOKEN_EXPIRY_TIME = 24 * 7  # 7 days
+
+# URLs for the front-end
+API_URL = env("VITE_BACKEND_URL", default="https://app.coord.dev")
```
Comment (JannKleen) [inline backend/config/settings/production.py:227]: Could we instead define the defaults in base.py or will that break the dev setup?

File: backend/config/urls.py [hunk 2]
```diff
@@ -6,17 +6,24 @@
 from django.views.generic import TemplateView
 from drf_spectacular.views import SpectacularAPIView, SpectacularSwaggerView
 
+home = TemplateView.as_view(
+    template_name="pages/home.html",
+    extra_context={
+        "api_url": settings.API_URL if hasattr(settings, "API_URL") else "",
```
Comment (JannKleen) [inline backend/config/urls.py:12]: no need for `hasattr` anymore now

File: backend/coordnet/templates/pages/home.html [hunk 3]
```diff
@@ -9,6 +9,13 @@
           href="{% static "coordination-network-circle-small.png" %}" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>coordination.network</title>
+    <script>
+      window.__COORDNET_CONFIG__ = {
+        apiUrl: "{{ api_url }}",
```
Comment (JannKleen) [inline backend/coordnet/templates/pages/home.html:14]: This will probably escape any `<>"'&`, but that shouldn't be an issue, right?

**Review by JannKleen — CHANGES_REQUESTED** [review]

File: backend/coordnet/templates/pages/home.html (hunk 3 above)
Comment (betamax) [reply backend/coordnet/templates/pages/home.html:14]: I think that's fine

File: backend/config/urls.py [hunk 4]
```diff
@@ -6,17 +6,24 @@
 from django.views.generic import TemplateView
 from drf_spectacular.views import SpectacularAPIView, SpectacularSwaggerView
 
+home = TemplateView.as_view(
+    template_name="pages/home.html",
+    extra_context={
+        "api_url": settings.API_URL if settings.API_URL else "",
```
Comment (JannKleen) [inline backend/config/urls.py:12]: Sorry, I wasn't very precise, settings will always have the values, there should be no need for a fallback.

**Review by JannKleen — CHANGES_REQUESTED** [review]

**Review by JannKleen — APPROVED** [review]

**Pull request merged**